### PR TITLE
Make the text charcoal for 'change your focus area'

### PR DESCRIPTION
**Pull request by mehalshah**

### CODE REVIEW

File: code-studio/src/js/components/progress/course_progress_row.jsx [hunk 1]
```diff
@@ -62,7 +62,7 @@ const styles = {
   },
   changeFocusArea: {
     fontFamily: '"Gotham 5r", sans-serif',
-    color: color.black,
+    color: color.dark_charcoal,
```
Comment (joshlory) [inline code-studio/src/js/components/progress/course_progress_row.jsx:65]: I think what we really want is the [default text color `#333`](https://github.com/code-dot-org/code-dot-org/blob/b3794b7e6ab42375d2a922c7f4e02d70c783397a/dashboard/app/assets/stylesheets/scaffolds.css.scss#L3).

**Comment by mehalshah** [comment]
Charcoal
image

Default Color
image

@tanyaparker and @breville - opinions?

**Comment by breville** [comment]
i like the change to charcoal

**Comment by joshlory** [comment]
All the other default text on the site is `#333`.  Any reason to make this specific text `#4d575f`?

Most monitors won't be able to tell the difference.

**Comment by tanyaparker** [comment]
I think we should basically change everywhere we have black and #333 to dark charcoal. I didn't comment on it before because it's not high pri, but it stood out to me a lot more on this particular text.

**Comment by tanyaparker** [comment]
#333 is not the worst for level titles and level text. But I think it's a little too dark.

**Comment by mehalshah** [comment]
So whats the deal here - should I 1) make the change to charcoal and 2) DTP with it?

**Comment by breville** [comment]
Changing everything is apparently out of scope right now, so suggestion is to just change this one piece of text to charcoal and note that we'd like to change the rest to charcoal later.

**Comment by tanyaparker** [comment]
Exactly ^

**Pull request merged**